assets/minecraft/lang/ru_ru.json

```
@@ -311,8 +311,8 @@
   "unlocks.unlock.amphibian.name": "Ихтиандр",
   "unlocks.unlock.apprentice_shoveler.description": "Working your way up in the shoveling business",
   "unlocks.unlock.apprentice_shoveler.name": "Apprentice Shoveler",
-  "unlocks.unlock.archer.description": "We would've called this unlock sagittarii but someone said it sounded weird and that no one speaks Latin anymore",
-  "unlocks.unlock.archer.name": "Archer",
+  "unlocks.unlock.archer.description": "Мы бы назвали это улучшение «Лучник», но это название уже занято другим улучшением",

```

Думаю, можно без потерь убрать лишнее
```suggestion
  "unlocks.unlock.archer.description": "Мы бы назвали его «Лучник», но название оказалось занято",
```